ngraph/frontend/tensorflow/include/tensorflow_frontend/decoder.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/variant.hpp>
+
+namespace ngraph {
+namespace frontend {
+
+class DecoderBase {
```

Comment:
shouldn't this class be the base for all frontends and be in the common frontend directory?

Reply:
it should in the future but it is planned for separate activity. we have dedicated ticket for this.